client/android/dns_list.go

```diff
@@ -0,0 +1,21 @@
+package android
+
+// DNSList is a wrapper of []string
+type DNSList struct {
+	items []string
+}
+
+// Add new DNS address to the collection
+func (array *DNSList) Add(s string) {
+	array.items = append(array.items, s)
+}
+
+// Get return an element of the collection
+func (array *DNSList) Get(i int) string {
+	return array.items[i]
```

Comment:
We can get panic here, better to check the range of i.

Reply:
Thank you! I pushed the fix and I also updated the Android code